common/src/main/cpp/extensions/openxr_fb_scene_capture_extension_wrapper.cpp

```diff
@@ -96,16 +96,38 @@ void OpenXRFbSceneCaptureExtensionWrapper::_on_instance_destroyed() {
 	cleanup();
 }
 
-bool OpenXRFbSceneCaptureExtensionWrapper::request_scene_capture() {
-	XrAsyncRequestIdFB requestId;
+bool OpenXRFbSceneCaptureExtensionWrapper::request_scene_capture(String p_request, SceneCaptureCompleteCallback p_callback, void *p_userdata) {
+	if (scene_capture_enabled) {
+		ERR_PRINT("Already running scene capture");
+		p_callback(XR_ERROR_VALIDATION_FAILURE, p_userdata);
+		return false;
+	}
+
+	XrAsyncRequestIdFB request_id;
+
+	CharString cstr = p_request.ascii();
+
 	XrSceneCaptureRequestInfoFB request;
 	request.type = XR_TYPE_SCENE_CAPTURE_REQUEST_INFO_FB;
 	request.next = nullptr;
-	request.requestByteCount = 0;
-	request.request = nullptr;
-	XrResult result = xrRequestSceneCaptureFB((XrSession)get_openxr_api()->get_session(), &request, &requestId);
-	scene_capture_enabled = (result == XR_SUCCESS);
-	return scene_capture_enabled;
+	request.requestByteCount = cstr.size();
+	request.request = cstr.ptr();
```

Comment:
Can you update this initialization to braced-initialization.

Reply:
Certainly! Updated in my latest push